Sources/Sentry/Public/SentryOptions.h

```diff
@@ -157,6 +157,13 @@ NS_SWIFT_NAME(Options)
  */
 @property (nullable, nonatomic, copy) SentryBeforeSendSpanCallback beforeSendSpan NS_SWIFT_SENDABLE;
 
+/**
+ * When enabled, the SDK sends logs to Sentry. Logs can be captured using the @c SentrySDK.logger
+ * API, which provides structured logging with attributes.
+ * @note Default value is @c NO .
+ */
+@property (nonatomic, assign) BOOL enableLogs;
```

Comment:
`h`: Please add a test to SentryOptionsTest, similar to for example `testEnableNetworkBreadcrumbs`. This test should then fail, because you didn't add this new flag to `SentryOptionsInternal.validateOptions`.